csrc/mmdeploy/device/cuda/cuda_device.cpp

```diff
@@ -69,7 +69,7 @@ class CudaDeviceMemory : public NonCopyable {
  public:
   explicit CudaDeviceMemory(int device_id) : device_id_(device_id), size_(), owned_block_() {}
   Result<void> Init(size_t size, Allocator allocator, size_t alignment, uint64_t flags) {
-    if (alignment != 1) {
+    if (alignment > 256 || 256 % alignment != 0) {
```

Comment:
How about giving a  default value,  caller does not know moderate `alignment` value.